CONTRIBUTING.md

```diff
@@ -36,52 +38,55 @@ Every Issue and PR should be tagged with one or more of the following labels to
+  - Project Name = Sub-project name.
+- Issues must be correctly set with the right labels and Milestone by the sub-project lead.
   - Relevant Milestone for the upcoming release (M1 - M6).
 - Contributors can assign an unassigned issue to themselves if they would like to work on them
-- Workstream Leads have the responsibility to assign stalling or priority issues to individuals to address them.
-- WSL may label an issue as "**Returned**" to request further information or to decide not to work on it.
+- Sub-project leads have the responsibility to assign stalling or priority issues to individuals to address them.
+- Sub-project leads may label an issue as "**Returned**" to request further information or to decide not to work on it.
 
 <a name="prs"></a>
 ### Pull Requests
 - PRs dealing with minor editorial changes can be raised without a corresponding Issue, but must be labelled as "Minor". Otherwise, PRs must only be created when there is an issue present and a decision to create a PR is made. 
 - A PR must reference the issue it is resolving in the description field.
    - using GitHub predefined keyword "Fixes Issue#" will automatically close the issue which the PR is resolving when it is merged.
-- A PR for an issue must only be created by the contributor assigned to the issue (either by self assignment or by the respective WSL).
+- A PR for an issue must only be created by the contributor assigned to the issue (either by self assignment or by the respective sub-project lead).
 - Only the person assigned ("assignee") to a PR may edit it. The normal process is for others to make suggests in "Comments" but not directly edit a PR unless the assignee has granter another contributor permission to contribute directly to the PR.
-- A PR is recommended to only create/modify content within the scope of a single workstream.
+- A PR is recommended to only create/modify content within the scope of a single chapter in a single sub-project if possible.
 - A PR should only add/change content related to the issue associated with the PR.
 - Comments/sub-Conversations within a PR may only be marked Resolved by:
   - A) The originator of the sub-conversation. 
-  - B) The relevant WSL or 
-  - C) An automated timeout of 48hrs when there has been no additional comments after an update is posted to resolve the sub-thread. 
+  - B) The relevant sub-project lead or 
+  - C) An automated timeout of 1 week when there has been no additional comments after an update is posted to resolve the sub-thread. 
   - **The person responding to a sub-conversation shall not mark it as Resolved**.
 - All changes to an existing PRs shall be made through subsequent commits. 
   - **Do not modify an existing commit, create separate commits under the same PR**.
+  - After major changes to a PR reapproval is needed from all previous approvers
   - PRs will be merged using "Squash and Merge" to enforce a linear commit history.
 
```

Comment:
can we add that the person performing the merge shall ensure that the branch is deleted

Reply:
to be honest I would like to add a bullet point saying that PR-s should be created from forks instead from the upstream repo. In this way the upstream repo would not be polluted with development branches.

Reply:
Agree with Gergely here.

Reply:
because of permission issues at AT&T I was never able to fork. I haven't tried at MSFT as am waiting for GitLab conversion.

Reply:
> because of permission issues at AT&T I was never able to fork. I haven't tried at MSFT as am waiting for GitLab conversion.

Maybe we can discuss this on the TSC. If it is not mandatory for you to open pr-s from MSFT-s GitHub org you can simply use your personal GitHub org.

Reply:
> Also, there are minor updates that are made during some meetings and approved in it.

Sorry, I do not understand this comment. Can you please explain?

Reply:
@CsatariGergely This is with respect to forks -- what I am pointing out is that in many sub-project meetings we have made a quick change (typo, grammar, etc.) and approved it all during the same meeting. Creating a fork etc. would be a burden in such circumstances.